doc/index.md

```diff
@@ -64,37 +122,58 @@ Cqueues can be integrated with almost any main loop or event library you may enc
 
 All connection types expose the fields:
 
+
 ### `connection.type` <!-- --> {#connection.type}
 
-Either `"client"` or `"server"`
+The mode of use for the connection object. Valid values are:
+- `"client"` - Connects to a remote uri
+- `"server"` - Listens for connection on a local uri
 
 
 ### `connection.version` <!-- --> {#connection.version}
 
-The HTTP version as a number
+The HTTP version number of the connection. 
```

Comment:
Important to note whether it's a number or a string containing a number. i.e. `1.1` vs `"1.1"`